docs/dev/table/functions.md

```diff
@@ -2614,6 +2626,18 @@ STRING.rtrim()
         <p>E.g., <code>'This is a test String. '.rtrim()</code> returns "This is a test String.".</p>
       </td>
     </tr>
+
+    <tr>
+      <td>
+        {% highlight java %}
+STRING.repeat(INT)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns a string that repeats the base <i>STRING</i> <i>INT</i> times.</p> 
+        <p>E.g., <code>"This is a test String. ".repeat(2)</code> returns "This is a test String.This is a test String.".</p>
```

Comment:
There's an additional space after the full stop. (I've fixed it in my branch, so just let it be there.)

Reply:
Thank you very much. @xccui